core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -93,7 +93,7 @@ angular.module('oppia').directive('aboutPage', [
               .slice(0, listOfNamesToThank.length - 1).join(', ') +
               ' & ' + listOfNamesToThank[listOfNamesToThank.length - 1];
             ctrl.aboutPageMascotImgUrl = UrlInterpolationService
-              .getStaticImageUrl('/general/about_page_mascot.png');
+              .getStaticImageUrl('/general/about_page_mascot.webp');
```

Comment:
Will this file work with browsers that do not support WebP?

Reply:
replaced the variable with the image itself in the html